node-typescript/push-notification-with-fcm/src/utils.ts

```diff
@@ -0,0 +1,29 @@
+import admin from 'firebase-admin';
+
+/**
+ * Throws an error if any of the keys are missing from the object
+ * @param {*} obj
+ * @param {string[]} keys
+ * @throws {Error}
+ */
+export function throwIfMissing(obj: any, keys: string[]) {
+  const missing = [];
+  for (let key of keys) {
+    if (!(key in obj) || !obj[key]) {
+      missing.push(key);
+    }
+  }
+  if (missing.length > 0) {
+    throw new Error(`Missing required fields: ${missing.join(', ')}`);
+  }
+}
+
+/**
+ * @param {admin.messaging.Message} payload
+ * @returns {Promise<string>}
+ */
+export async function sendPushNotification(
```

Comment:
Let's please copy existing logic from Node version of this function, for sick of consistency: https://github.com/appwrite/templates/blob/main/node/push-notification-with-fcm/src/utils.js

Reply:
I tried to do less changes and wrote this code as similar to node. if do futher improvement like node then it will give types errors. Please support and inform me next.